locale/fi/LC_MESSAGES/django.po

```diff
@@ -2520,16 +2541,17 @@ msgid "Download"
 msgstr "Lataa"
 
 #: exercise/templates/exercise/staff/_submissions_table.html:37
-#, python-format
 msgid ""
 "\n"
 "    %(limit)s latest submissions.\n"
-"    <a class=\"aplus-button--secondary aplus-button--sm\" href=\"%(url)s\">Show all</a>\n"
+"    <a class=\"aplus-button--secondary aplus-button--xs\" role=\"button\" "
```

Comment:
Now I see that the Finnish string does not have the `role="button"` like the English one has. You probably didn't change this string yourself, but `manage.py makemessages` decided that the line was too long and it divided it to multiple lines.

Reply:
It seems like the file was not updated after it was added here.

Reply:
Can you rebase on top of current master and rerun makemessages?

Reply:
I can, but that is the last thing I can do for now 😁 So hope it is otherwise good

Reply:
So they both should have the "role=button" or neither should have?

Reply:
Oops actually I can't do that because I just wiped my work computer

Reply:
Jaakko fixed that string in a separate commit in master, so we will need to rebase this PR onto master and remove the conflicting changes in django.po. Sounds like it will be my task next week!